frontend/src/queries/nodes/DataVisualization/Components/Variables/addVariableLogic.ts

```diff
@@ -97,20 +98,18 @@ export const addVariableLogic = kea<addVariableLogicType>([
             },
         ],
     }),
-    loaders(({ values, actions }) => ({
-        savedVariable: [
```

Comment:
Double checking: this variable isn't used

Reply:
Nope - not used 😄